wcfsetup/install/files/lib/data/DatabaseObject.class.php

```diff
@@ -41,6 +41,24 @@
 	 */
 	protected static $sortOrder = null;
 	
+	/**
+	 * The list of derived database table names based on the class name.
+	 * 
+	 * WARNING: This is strictly an internal lookup table. DO NOT USE IT.
```

Comment:
As you are using an array now anyway: You *could* leave this as a static variable inside the method.

Reply:
Could be worth it, bonus points for hiding the internals, but I haven't tried it. Did you check if it behaves properly with derived classes, especially given a random initialization order?

Reply:
Looking good: https://3v4l.org/gefrF.

See also this top comment: http://php.net/manual/en/language.variables.scope.php#59676